cmd/get/clusters/flag.go

```diff
@@ -0,0 +1,25 @@
+package clusters
+
+import (
+	"github.com/spf13/cobra"
+	"k8s.io/cli-runtime/pkg/genericclioptions"
+)
+
+type flag struct {
+	config genericclioptions.RESTClientGetter
+	print  *genericclioptions.PrintFlags
+}
+
+func (f *flag) Init(cmd *cobra.Command) {
+	f.config = genericclioptions.NewConfigFlags(true)
+	f.print = genericclioptions.NewPrintFlags("")
```

Comment:
Adding kubectl-specific flags, for printing (such as `--output`) and for basic configuration (such as `--context`)